package.json

```diff
@@ -35,9 +35,11 @@
     "react-dom": "^0.14.x "
   },
   "devDependencies": {
-    "babel": "^5.x.x",
-    "babel-core": "^5.x.x",
-    "babel-loader": "^5.x.x",
+    "babel": "~6.0.15",
+    "babel-core": "~6.0.15",
+    "babel-loader": "~6.0.1",
+    "babel-preset-es2015": "~6.0.15",
+    "babel-preset-react": "~6.0.15",
```

Comment:
Let's stay consistent with the other deps and use `^` for now instead of `~`. These prefixes can be controversial, if you think they should change, let's open a new issue for it.